core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -54,6 +54,9 @@ export class CkEditorInitializerService {
       rteHelperService: RteHelperService,
       htmlEscaperService: HtmlEscaperService,
       contextService: ContextService, ngZone: NgZone): void {
+    if (rteHelperService === undefined) {
```

Comment:
Just to check, when does this occur?

Reply:
@srijanreddy98 Can you please answer this?

Reply:
This will occur in pages that don't use angularjs and this case will stay until rte-helper service hasn't been migrated. In the next week we should see this if block be removed completely (as we are migrating it next).